index.html

```diff
@@ -802,10 +802,13 @@ <h3>Presentations</h3>
 A <a>verifiable presentation</a> expresses data from one or more
 <a>verifiable credentials</a>, and is packaged in such a way that the
 authorship of the data is <a>verifiable</a>. If <a>verifiable credentials</a>
-are presented directly, they become <a>verifiable presentations</a>. Data
-formats derived from <a>verifiable credentials</a> that are cryptographically
-<a>verifiable</a>, but do not of themselves contain
-<a>verifiable credentials</a>, might also be <a>verifiable presentations</a>.
+are presented directly, they become <a>verifiable presentations</a>.
```

Comment:
IMO, that sentence is in contradiction to 4.10 Presentations.

Reply:
Can you say more? I'm not seeing the contradiction.

Reply:
Can we just strike the sentence about VCs "becoming" VPs here? What does it do for us? I think the language here was trying to be helpful or inclusive about something that was fuzzily defined in the past that's no longer needed now.